packages/common/src/hardforks/shanghai.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "shanghai",
-  "comment": "Next feature hardfork after the merge hardfork",
+  "comment": "Next feature hardfork after the merge hardfork having withdrawals, warm coinbase, push0, limit/meter initcode",
```

Comment:
Could we add that this fork is still experimental?

Reply:
Yes, that would be important so people can clearly distinguish, we should definitely do before next releases.

Have added to tiny-fixes issue: https://github.com/ethereumjs/ethereumjs-monorepo/issues/2105